test/APINotes/Inputs/custom-frameworks/APINotesFrameworkTest.framework/Headers/APINotesFrameworkTest.apinotes

```diff
@@ -39,6 +39,27 @@ SwiftVersions:
   - Version: 3.0
     Classes:
       - Name: TestProperties
+        Methods:
+          - Selector: accessorsOnlyRenamedRetyped
+            MethodKind: Instance
+            SwiftName: 'renamedAndRetyped()'
+            ResultType: 'id _Nonnull'
```

Comment:
Sure, but the opposite is also true—if mismatched types declared in the header file don't work, that's also a bug.